doc/core/pending_api_cleanups.md

```diff
@@ -0,0 +1,15 @@
+There are times when we make changes that include a temporary shim for
```

Comment:
Why bump the *minor* version number when making breaking changes, instead of the major number as semantic versioning requires? That messes with users of dependency managers that understand semantic versioning.